.travis.yml

```diff
@@ -20,8 +20,23 @@ env:
   - DISTRIB="conda" PYTHON_VERSION="3.6" COVERAGE="true" SKLEARN_VERSION="0.18.2"
   - DISTRIB="conda" PYTHON_VERSION="3.6" EXAMPLES="true" SKLEARN_VERSION="0.18.2"
   - DISTRIB="conda" PYTHON_VERSION="3.6" DOCTEST="true" SKLEARN_VERSION="0.18.2"
+  - DISTRIB="conda" PYTHON_VERSION="3.6" DOCPUSH="true" SKLEARN_VERSION="0.18.2"
 #  - DISTRIB="conda" PYTHON_VERSION="3.7" SKLEARN_VERSION="0.18.2"
 
 install: source ci_scripts/install.sh
 script: bash ci_scripts/test.sh
-after_success: source ci_scripts/success.sh
+after_success: source ci_scripts/success.sh && source ci_scripts/create_doc.sh $TRAVIS_BRANCH "doc_result"
+
+# travis will check the deploy on condition, before actually running before_deploy
+# before_deploy: source ci_scripts/create_doc.sh $TRAVIS_BRANCH "doc_result"
+
+deploy:
```

Comment:
Could you please add a link to https://docs.travis-ci.com/user/deployment/pages/ so that future developers know where to read about the travis-ci features we're using here?